msgid ""
msgstr ""
"Project-Id-Version: red-discordbot\n"
"POT-Creation-Date: 2022-07-19 15:11+0000\n"
"Last-Translator: \n"
"Language-Team: Russian\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: redgettext 3.4.2\n"
"Plural-Forms: nplurals=4; plural=((n%10==1 && n%100!=11) ? 0 : ((n%10 >= 2 && n%10 <=4 && (n%100 < 12 || n%100 > 14)) ? 1 : ((n%10 == 0 || (n%10 >= 5 && n%10 <=9)) || (n%100 >= 11 && n%100 <= 14)) ? 2 : 3));\n"
"X-Crowdin-Project: red-discordbot\n"
"X-Crowdin-Project-ID: 289505\n"
"X-Crowdin-Language: ru\n"
"X-Crowdin-File-ID: 688\n"
"Language: ru_RU\n"

#: redbot/cogs/audio/core/utilities/formatting.py:74
msgid "tracks"
msgstr "треки"

#: redbot/cogs/audio/core/utilities/formatting.py:84
#: redbot/cogs/audio/core/utilities/formatting.py:272
msgid "Page {page_num}/{total_pages}"
msgstr "Страница {page_num}/{total_pages}"

#: redbot/cogs/audio/core/utilities/formatting.py:95
#: redbot/cogs/audio/core/utilities/playlists.py:526
msgid "Connection to Lavalink node has failed"
msgstr ""

#: redbot/cogs/audio/core/utilities/formatting.py:98
#: redbot/cogs/audio/core/utilities/player.py:589
#: redbot/cogs/audio/core/utilities/playlists.py:529
msgid "Please check your console or logs for details."
msgstr "Пожалуйста, проверьте вашу консоль или логи для получения более подробной информации."

#: redbot/cogs/audio/core/utilities/formatting.py:106
#: redbot/cogs/audio/core/utilities/playlists.py:561
msgid "Connect to a voice channel first."
msgstr "Сначала подключитесь к голосовому каналу."

#: redbot/cogs/audio/core/utilities/formatting.py:109
#: redbot/cogs/audio/core/utilities/playlists.py:554
msgid "Connection to Lavalink node has not yet been established."
msgstr ""

#: redbot/cogs/audio/core/utilities/formatting.py:116
msgid "Unable To Play Tracks"
msgstr "Не удается воспроизвести треки"

#: redbot/cogs/audio/core/utilities/formatting.py:116
#: redbot/cogs/audio/core/utilities/player.py:441
#: redbot/cogs/audio/core/utilities/player.py:522
msgid "Queue size limit reached."
msgstr "Достигнут предел размера очереди."

#: redbot/cogs/audio/core/utilities/formatting.py:154
#: redbot/cogs/audio/core/utilities/player.py:597
msgid "Track Enqueued"
msgstr "Трек поставлен в очередь"

#: redbot/cogs/audio/core/utilities/formatting.py:168
#: redbot/cogs/audio/core/utilities/player.py:546
msgid "This track is not allowed in this server."
msgstr "Этот трек не разрешен на этом сервере."

#: redbot/cogs/audio/core/utilities/formatting.py:185
#: redbot/cogs/audio/core/utilities/player.py:568
msgid "Track exceeds maximum length."
msgstr "Этот трек превышает максимальную длину."

#: redbot/cogs/audio/core/utilities/formatting.py:200
#: redbot/cogs/audio/core/utilities/player.py:600
msgid "{time} until track playback: #{position} in queue"
msgstr "{time} до воспроизведения трека: #{position} в очереди"

#: redbot/cogs/audio/core/utilities/formatting.py:260
msgid "Tracks Found:"
msgstr "Найденные треки:"

#: redbot/cogs/audio/core/utilities/formatting.py:261
msgid "search results"
msgstr "результаты поиска"

#: redbot/cogs/audio/core/utilities/formatting.py:263
msgid "Folders Found:"
msgstr "Найденные папки:"

#: redbot/cogs/audio/core/utilities/formatting.py:264
msgid "local folders"
msgstr "локальные папки"

#: redbot/cogs/audio/core/utilities/formatting.py:266
msgid "Files Found:"
msgstr "Найденные файлы:"

#: redbot/cogs/audio/core/utilities/formatting.py:267
msgid "local tracks"
msgstr "локальные треки"

#: redbot/cogs/audio/core/utilities/formatting.py:379
#: redbot/cogs/audio/core/utilities/playlists.py:240
msgid "Unknown"
msgstr "Неизвестный"

#: redbot/cogs/audio/core/utilities/formatting.py:380
msgid " - Name:   <{pname}>\n"
" - Scope:  < {scope} >\n"
" - ID:     < {pid} >\n"
" - Tracks: < {ptracks} >\n"
" - Author: < {author} >\n\n"
msgstr " - Имя: <{pname}>\n"
" - Длина: < {scope} >\n"
" - ID: < {pid} >\n"
" - Треки: < {ptracks} >\n"
" - Автор: < {author} >\n\n"

#: redbot/cogs/audio/core/utilities/local_tracks.py:109
#: redbot/cogs/audio/core/utilities/player.py:235
#: redbot/cogs/audio/core/utilities/player.py:245
#: redbot/cogs/audio/core/utilities/player.py:327
#: redbot/cogs/audio/core/utilities/player.py:648
msgid "Invalid Environment"
msgstr "Неверное окружение"

#: redbot/cogs/audio/core/utilities/local_tracks.py:109
msgid "No localtracks folder."
msgstr "Нет локальной папки треков."

#: redbot/cogs/audio/core/utilities/miscellaneous.py:50
msgid "Not enough {currency}"
msgstr "Недостаточно {currency}"

#: redbot/cogs/audio/core/utilities/miscellaneous.py:51
msgid "{required_credits} {currency} required, but you have {bal}."
msgstr "{required_credits} {currency} требуется, у вас {bal}."

#: redbot/cogs/audio/core/utilities/player.py:78
msgid "music in {} servers"
msgstr "музыка на {} серверах"

#: redbot/cogs/audio/core/utilities/player.py:136
#: redbot/cogs/audio/core/utilities/player.py:140
#: redbot/cogs/audio/core/utilities/player.py:145
msgid "There's nothing in the queue."
msgstr "Очередь пуста."

#: redbot/cogs/audio/core/utilities/player.py:142
msgid "Currently livestreaming {track}"
msgstr "Сейчас в прямом эфире {track}"

#: redbot/cogs/audio/core/utilities/player.py:147
msgid "{time} left on {track}"
msgstr "{time} осталось в {track}"

#: redbot/cogs/audio/core/utilities/player.py:155
#: redbot/cogs/audio/core/utilities/player.py:190
msgid "Track Skipped"
msgstr "Трек пропущен"

#: redbot/cogs/audio/core/utilities/player.py:168
msgid "Track number must be equal to or greater than 1."
msgstr "Номер трека должен быть больше или равен 1."

#: redbot/cogs/audio/core/utilities/player.py:174
msgid "There are only {queuelen} songs currently queued."
msgstr "В настоящее время в очереди стоят только песни: {queuelen}."

#: redbot/cogs/audio/core/utilities/player.py:180
msgid "{skip_to_track} Tracks Skipped"
msgstr "{skip_to_track} треков пропущено"

#: redbot/cogs/audio/core/utilities/player.py:236
msgid "The owner needs to set the Spotify client ID and Spotify client secret, before Spotify URLs or codes can be used. \n"
"See `{prefix}audioset spotifyapi` for instructions."
msgstr "Владелец должен установить идентификатор клиента Spotify и секрет клиента Spotify, прежде чем можно будет использовать URL-адреса или коды Spotify..\n"
"Просмотрите `{prefix}audioset spotifyapi` для получения инструкций."

#: redbot/cogs/audio/core/utilities/player.py:246
msgid "The owner needs to set the YouTube API key before Spotify URLs or codes can be used.\n"
"See `{prefix}audioset youtubeapi` for instructions."
msgstr "Владелец должен установить ключ YouTube API перед использованием URL-адресов или кодов Spotify.\n"
"Смотрите `{prefix}audioset youtubeapi` для получения инструкций."

#: redbot/cogs/audio/core/utilities/player.py:255
#: redbot/cogs/audio/core/utilities/player.py:364
#: redbot/cogs/audio/core/utilities/playlists.py:594
msgid "Unable To Get Tracks"
msgstr "Не удается получить треки"

#: redbot/cogs/audio/core/utilities/player.py:256
#: redbot/cogs/audio/core/utilities/player.py:365
#: redbot/cogs/audio/core/utilities/playlists.py:595
msgid "Wait until the playlist has finished loading."
msgstr "Дождитесь окончания загрузки плейлиста."

#: redbot/cogs/audio/core/utilities/player.py:267
#: redbot/cogs/audio/core/utilities/player.py:309
#: redbot/cogs/audio/core/utilities/player.py:407
#: redbot/cogs/audio/core/utilities/playlists.py:605
#: redbot/cogs/audio/core/utilities/playlists.py:638
msgid "Nothing found."
msgstr "Ничего не найдено."

#: redbot/cogs/audio/core/utilities/player.py:270
#: redbot/cogs/audio/core/utilities/player.py:311
#: redbot/cogs/audio/core/utilities/player.py:422
#: redbot/cogs/audio/core/utilities/playlists.py:607
#: redbot/cogs/audio/core/utilities/playlists.py:640
msgid "Track is not playable."
msgstr "Трек не воспроизводится."

#: redbot/cogs/audio/core/utilities/player.py:271
#: redbot/cogs/audio/core/utilities/player.py:312
#: redbot/cogs/audio/core/utilities/player.py:424
#: redbot/cogs/audio/core/utilities/playlists.py:608
#: redbot/cogs/audio/core/utilities/playlists.py:641
msgid "**{suffix}** is not a fully supported format and some tracks may not play."
msgstr "**{suffix}** не является полностью поддерживаемым форматом, и некоторые треки не могут воспроизводиться."

#: redbot/cogs/audio/core/utilities/player.py:301
#: redbot/cogs/audio/core/utilities/player.py:392
#: redbot/cogs/audio/core/utilities/player.py:655
#: redbot/cogs/audio/core/utilities/playlists.py:425
#: redbot/cogs/audio/core/utilities/playlists.py:626
#: redbot/cogs/audio/core/utilities/playlists.py:655
msgid "Unable to Get Track"
msgstr "Не удается получить трек"

#: redbot/cogs/audio/core/utilities/player.py:302
#: redbot/cogs/audio/core/utilities/playlists.py:426
msgid "I'm unable to get a track from the Lavalink node at the moment, try again in a few minutes."
msgstr ""

#: redbot/cogs/audio/core/utilities/player.py:328
msgid "The Spotify API key or client secret has not been set properly. \n"
"Use `{prefix}audioset spotifyapi` for instructions."
msgstr "API ключ Spotify или секрет клиента были установлены неправильно. \n"
"Для получения инструкций используйте `{prefix}audioset spotifyapi`."

#: redbot/cogs/audio/core/utilities/player.py:352
msgid "Unable To Find Tracks"
msgstr "Не удается найти треки"

#: redbot/cogs/audio/core/utilities/player.py:353
msgid "This doesn't seem to be a supported Spotify URL or code."
msgstr "Это не похоже на поддерживаемый $SPOTIFY URL или код."

#: redbot/cogs/audio/core/utilities/player.py:379
msgid "{query} is not an allowed query."
msgstr "{query} не является допустимым запросом."

#: redbot/cogs/audio/core/utilities/player.py:393
#: redbot/cogs/audio/core/utilities/playlists.py:627
#: redbot/cogs/audio/core/utilities/playlists.py:656
msgid "I'm unable to get a track from Lavalink node at the moment, try again in a few minutes."
msgstr ""

#: redbot/cogs/audio/core/utilities/player.py:415
msgid "Local tracks will not work if the `Lavalink.jar` cannot see the track.\n"
"This may be due to permissions or because Lavalink.jar is being run in a different machine than the local tracks."
msgstr "Локальные треки не будут работать, если `Lavalink.jar` не будет видеть трек.\n"
"Это может быть вызвано правами или потому что Lavalink.jar запускается на машине, отличной от локальных треков."

#: redbot/cogs/audio/core/utilities/player.py:485
msgid " {bad_tracks} tracks cannot be queued."
msgstr " {bad_tracks} треков не могут быть поставлены в очередь."

#: redbot/cogs/audio/core/utilities/player.py:491
msgid "No Title"
msgstr "Без заголовка"

#: redbot/cogs/audio/core/utilities/player.py:497
msgid "Playlist Enqueued"
msgstr "Плейлист поставлен в очередь"

#: redbot/cogs/audio/core/utilities/player.py:500
msgid "Added {num} tracks to the queue.{maxlength_msg}"
msgstr "{num} треков добавлено в очередь.{maxlength_msg}"

#: redbot/cogs/audio/core/utilities/player.py:506
msgid "{time} until start of playlist playback: starts at #{position} in queue"
msgstr "{time} до начала воспроизведения плейлиста: начинается с #{position} в очереди"

#: redbot/cogs/audio/core/utilities/player.py:586
msgid "Nothing found"
msgstr "Ничего не найдено"

#: redbot/cogs/audio/core/utilities/player.py:621
msgid "Please wait, finding tracks..."
msgstr "Пожалуйста, подождите, поиск треков..."

#: redbot/cogs/audio/core/utilities/player.py:627
msgid "Getting track {num}/{total}..."
msgstr "Получение трека {num}/{total}..."

#: redbot/cogs/audio/core/utilities/player.py:628
msgid "Matching track {num}/{total}..."
msgstr "Сопоставление трека {num}/{total}..."

#: redbot/cogs/audio/core/utilities/player.py:629
#: redbot/cogs/audio/core/utilities/playlists.py:341
#: redbot/cogs/audio/core/utilities/playlists.py:414
msgid "Loading track {num}/{total}..."
msgstr "Загрузка трека {num}/{total}..."

#: redbot/cogs/audio/core/utilities/player.py:630
msgid "Approximate time remaining: {seconds}"
msgstr "Примерно времени осталось: {seconds}"

#: redbot/cogs/audio/core/utilities/player.py:656
msgid "I'm unable to get a track from Lavalink at the moment, try again in a few minutes."
msgstr "На данный момент я не могу получить трек от Lava_link, попробуйте еще раз через несколько минут."

#: redbot/cogs/audio/core/utilities/player.py:665
msgid "The connection was reset while loading the playlist."
msgstr "Соединение было сброшено при загрузке плейлиста."

#: redbot/cogs/audio/core/utilities/playlists.py:83
msgid "You do not have the permissions to manage {name} (`{id}`) [**{scope}**]."
msgstr "У вас нет прав для управления {name} (`{id}`) [**{scope}**]."

#: redbot/cogs/audio/core/utilities/playlists.py:101
msgid "You do not have the permissions to manage that playlist in {guild}."
msgstr "У вас нет прав для управления этим плейлистом в {guild}."

#: redbot/cogs/audio/core/utilities/playlists.py:108
msgid "You do not have the permissions to manage playlist owned by {user}."
msgstr "У вас нет прав на управление плейлистом, принадлежащим {user}."

#: redbot/cogs/audio/core/utilities/playlists.py:112
msgid "You do not have the permissions to manage playlists in {scope} scope."
msgstr "У вас нет прав на управление плейлистами в {scope} области."

#: redbot/cogs/audio/core/utilities/playlists.py:116
msgid "No access to playlist."
msgstr "Нет доступа к плейлисту."

#: redbot/cogs/audio/core/utilities/playlists.py:224
msgid "{match_count} playlists match {original_input}: Please try to be more specific, or use the playlist ID."
msgstr "{match_count} плейлисты соответствуют {original_input}: Пожалуйста, попробуйте более конкретно, или используйте ID плейлиста."

#: redbot/cogs/audio/core/utilities/playlists.py:241
msgid "{number}.    <{playlist.name}>\n"
" - Scope:  < {scope} >\n"
" - ID:     < {playlist.id} >\n"
" - Tracks: < {tracks} >\n"
" - Author: < {author} >\n\n"
msgstr "{number}. <{playlist.name}>\n"
" - Область: < {scope} >\n"
" - ID: < {playlist.id} >\n"
" - Треки: < {tracks} >\n"
" - Автор: < {author} >\n\n"

#: redbot/cogs/audio/core/utilities/playlists.py:258
msgid "{playlists} playlists found, which one would you like?"
msgstr "Найдено плейлистов: {playlists}, какой из них вы хотите?"

#: redbot/cogs/audio/core/utilities/playlists.py:277
#: redbot/cogs/audio/core/utilities/playlists.py:283
msgid "Too many matches found and you did not select which one you wanted."
msgstr "Найдено слишком много совпадений, и вы не указали, какой один из них вы хотели."

#: redbot/cogs/audio/core/utilities/playlists.py:308
msgid "Playlists you can access in this server:"
msgstr "Плейлисты, к которым вы можете получить доступ на этом сервере:"

#: redbot/cogs/audio/core/utilities/playlists.py:314
msgid "Playlists for {scope}:"
msgstr "Плейлисты для {scope}:"

#: redbot/cogs/audio/core/utilities/playlists.py:318
msgid "Page {page_num}/{total_pages} | {num} playlists."
msgstr "Страница {page_num}/{total_pages} | {num} плейлистов."

#: redbot/cogs/audio/core/utilities/playlists.py:334
#: redbot/cogs/audio/core/utilities/playlists.py:412
msgid "Please wait, adding tracks..."
msgstr "Пожалуйста, подождите, треки добавляются..."

#: redbot/cogs/audio/core/utilities/playlists.py:361
#: redbot/cogs/audio/core/utilities/playlists.py:464
msgid "Empty playlist {name} (`{id}`) [**{scope}**] created."
msgstr "Пустой плейлист {name} (`{id}`) [**{scope}**] создан."

#: redbot/cogs/audio/core/utilities/playlists.py:366
#: redbot/cogs/audio/core/utilities/playlists.py:469
msgid "Added {num} tracks from the {playlist_name} playlist. {num_bad} track(s) could not be loaded."
msgstr "Добавлено {num} треков из плейлиста {playlist_name}. Не удалось загрузить {num_bad} трек(ов)."

#: redbot/cogs/audio/core/utilities/playlists.py:371
#: redbot/cogs/audio/core/utilities/playlists.py:474
msgid "Added {num} tracks from the {playlist_name} playlist."
msgstr "Добавлено {num} треков из плейлиста {playlist_name}."

#: redbot/cogs/audio/core/utilities/playlists.py:375
#: redbot/cogs/audio/core/utilities/playlists.py:478
msgid "Playlist Saved"
msgstr "Плейлист сохранен"

#: redbot/cogs/audio/core/utilities/playlists.py:540
#: redbot/cogs/audio/core/utilities/playlists.py:553
#: redbot/cogs/audio/core/utilities/playlists.py:560
#: redbot/cogs/audio/core/utilities/playlists.py:571
msgid "Unable To Get Playlists"
msgstr "Невозможно получить плейлисты"

#: redbot/cogs/audio/core/utilities/playlists.py:541
msgid "I don't have permission to connect and speak in your channel."
msgstr "У меня нет разрешения подключаться и говорить на вашем канале."

#: redbot/cogs/audio/core/utilities/playlists.py:572
msgid "You must be in the voice channel to use the playlist command."
msgstr "Вы должны быть в голосовом канале, чтобы использовать команду плейлиста."

#: redbot/cogs/audio/core/utilities/playlists.py:680
msgid "the Global"
msgstr "глобальный"

#: redbot/cogs/audio/core/utilities/playlists.py:680
msgid "Global"
msgstr "Глобально"

#: redbot/cogs/audio/core/utilities/playlists.py:682
msgid "the Server"
msgstr "сервер"

#: redbot/cogs/audio/core/utilities/playlists.py:682
msgid "Server"
msgstr "Сервер"

#: redbot/cogs/audio/core/utilities/playlists.py:684
msgid "the User"
msgstr "пользователь"

#: redbot/cogs/audio/core/utilities/playlists.py:684
msgid "User"
msgstr "Пользователь"

#: redbot/cogs/audio/core/utilities/queue.py:40
msgid "__Too many songs in the queue, only showing the first 500__.\n\n"
msgstr "__Слишком много песен в очереди, отображаются только первые 500__.\n\n"

#: redbot/cogs/audio/core/utilities/queue.py:57
msgid "**Currently livestreaming:**\n"
msgstr "**Сейчас в прямом эфире:**\n"

#: redbot/cogs/audio/core/utilities/queue.py:59
#: redbot/cogs/audio/core/utilities/queue.py:64
msgid "Requested by: **{user}**"
msgstr "Запрошено: **{user}**"

#: redbot/cogs/audio/core/utilities/queue.py:62
msgid "Playing: "
msgstr "Воспроизводится: "

#: redbot/cogs/audio/core/utilities/queue.py:76
msgid "requested by **{user}**\n"
msgstr "по запросу: **{user}**\n"

#: redbot/cogs/audio/core/utilities/queue.py:80
msgid "Queue for __{guild_name}__"
msgstr "Очередь для __{guild_name}__"

#: redbot/cogs/audio/core/utilities/queue.py:88
msgid "Page {page_num}/{total_pages} | {num_tracks} tracks, {num_remaining} remaining\n"
msgstr "Страница {page_num}/{total_pages} | {num_tracks} треков, {num_remaining} осталось\n"

#: redbot/cogs/audio/core/utilities/queue.py:97
msgid "Auto-Play"
msgstr "Авто-воспроизведение"

#: redbot/cogs/audio/core/utilities/queue.py:103
msgid "Shuffle"
msgstr "Перемешивание"

#: redbot/cogs/audio/core/utilities/queue.py:109
msgid "Repeat"
msgstr "Повтор"

#: redbot/cogs/audio/core/utilities/queue.py:161
msgid "Matching Tracks:"
msgstr "Соответствующие треки:"

#: redbot/cogs/audio/core/utilities/queue.py:164
msgid "Page {page_num}/{total_pages} | {num_tracks} tracks"
msgstr "Страница {page_num}/{total_pages} | {num_tracks} треков"

